### PR TITLE
40.0.0

### PR DESCRIPTION
**Pull request by JamesRobertsonGames**
[PKG-2328]

- upstream: https://github.com/pypa/readme_renderer
- conda-forge: https://github.com/conda-forge/readme_renderer-feedstock
- new repo in our feedstocks

[PKG-2328]: https://anaconda.atlassian.net/browse/PKG-2328

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -11,34 +11,41 @@ source:
 
 build:
   number: 0
-  noarch: python
-  script: '{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv '
+  script: '{{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vvv '
```
Comment (lorepirri) [inline recipe/meta.yaml]:
```suggestion
  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
```

**Review by lorepirri — APPROVED** [review]
LGTM! Just the `build.script` needs a fix as per suggestion

File: recipe/meta.yaml [hunk 2]
```diff
@@ -11,34 +11,41 @@ source:
...
 
 requirements:
   host:
-    - python >=3.8
+    - python
     - pip
+    - wheel
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:20]: Missing `setuptools`

File: recipe/meta.yaml [hunk 3]
```diff
@@ -11,34 +11,41 @@ source:
...
 about:
   home: https://github.com/pypa/readme_renderer
   license: Apache-2.0
   license_family: APACHE
   license_file: LICENSE
-  summary: readme_renderer is a library for rendering "readme" descriptions for Warehouse
+  doc_url: https://github.com/pypa/readme_renderer
+  dev_url: https://github.com/pypa/readme_renderer
+  description: readme_renderer is a library for rendering readme descriptions for Warehouse
+  documentation: https://pypi.org/project/readme-renderer/
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]:
```suggestion
```
We don't require the `documentation` key

**Review by skupr-anaconda — APPROVED** [review]

**Pull request merged**